tests/Doctrine/Tests/Models/Cache/Token.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Doctrine\Tests\Models\Cache;
+
+/**
+ * @Entity
+ * @Cache("READ_ONLY")
```

Comment:
Can you try to set it as `NONSTRICT_READ_WRITE`? See http://doctrine-orm.readthedocs.org/en/latest/reference/second-level-cache.html#caching-mode